data_inconsistencies.py

```diff
@@ -32,6 +33,35 @@ def get_inconsistencies():
     return incons
 
 
+def inconsistencies_metric(file_name):
+    f = utils.get_all_images()
+    total_img = len(f)
+    with open(file_name, 'rU') as csvfile:
+        reader = csv.DictReader(csvfile)
+        data = {}
+        for row in reader:
+            for header, value in row.items():
+                try:
+                    data[header].append(value)
+                except KeyError:
+                    data[header] = [value]
+
+    firefox = data['FIREFOX']
+    n_incons = len(firefox)
+    chrome = data['CHROME']
+    incons_f = [int(x == "True") for x in firefox]
+    incons_c = [int(x == "True") for x in chrome]
+    print("Number of Photos: %d " % total_img)
+    print("Number of Pair of images: %d " % int((total_img-n_incons)/2))
```

Comment:
Shouldn't it be `(total_img - n_incons * 2) / 2`?

Reply:
Nit: here and in the following, put spaces between the math operators to make the formulas easier to read

Reply:
Total images - n_icons because in the photos which are inconsistent , there is no paired photo . So n_incons *2 would be wrong.